pkgs/tools/networking/oha/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchFromGitHub, lib, pkg-config, rustPlatform, stdenv, openssl, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "oha";
+  version = "0.4.6";
+
+  src = fetchFromGitHub {
+    owner = "hatoo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0vx8ki0wi9xil2iksvxzh8mhx4c5ikkhdcnc8mcwqn14cvjkggja";
+  };
+
+  cargoSha256 = "1nx2lvbjflsjma5q9ck6vq499hf75w91i4h8wlzr83wqk37i7rhc";
+
+  nativeBuildInputs = lib.optional stdenv.isLinux pkg-config;
+
+  buildInputs = lib.optional stdenv.isLinux openssl
+    ++ lib.optional stdenv.isDarwin Security;
+
+  # tests don't work inside the sandbox
```

Comment:
why? tests don't work inside the sandbox is too generic. If , for example, tests require internet connection then write it.

Reply:
[this mutex](https://github.com/hatoo/oha/blob/v0.4.6/tests/tests.rs#L6) consistently gets poisoned inside the sandbox and works otherwise. i don't know what is causing the issue but i'm sure it has something to do with sandboxing

Reply:
Are you on darwin? The sandbox there blocks local network access.

Reply:
I'm on nixos